salt/modules/deb_postgres.py

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
+'''
+Module to provide Postgres compatibility to salt for debian family specific tools.
+
+'''
+
+# Import python libs
+from __future__ import absolute_import
+import logging
+import pipes
+
+# Import salt libs
+import salt.utils
+
+# Import 3rd-party libs
+import salt.ext.six as six
+
+
+log = logging.getLogger(__name__)
+
+__virtualname__ = 'postgres'
```

Comment:
Uhm, i though one of the modules declaring the same __virtualname__ wins the name and then they are not merged together, but i can be wrong here.
Interresting, i'm testing this to verify.